src/test/java/mil/dds/anet/test/emails/AccountDeactivationWorkerTest.java

```diff
@@ -120,7 +115,7 @@ public void testWarnings() throws Exception {
   public void testDeactivation() throws Exception {
 
     // Configure
-    final Person testPersonEotActive = createDummyPerson(Instant.now().minus(1, ChronoUnit.DAYS),
+    final Person testPersonEotActive = createDummyPerson(Instant.now().minus(1, ChronoUnit.HOURS),
```

Comment:
Are any of the other changes to this file except this one actually needed?

Reply:
I thought the idea was to use this ticket to migrate this test to Junit5?